hubgrep/frontend_blueprint/templates/components/result_card/result_card.html

```diff
@@ -1,3 +1,11 @@
+{% macro repo_time_value(datetime) %}
+    {% if datetime %}
+        {{ timeago.format(datetime, datetime_now, get_locale()) }}
+    {% else %}
+        {{ constants.MISSING_VALUE_DEFAULT }}
+    {% endif %}
+{% endmacro %}
```

Comment:
This changes what we display a bit, and we should consider this in particular:

I changed search item dates in the list view to show a "humanized" value of "how long a go" (via the lib "timeago"), and I moved the exact date and time into the tooltip AND it's shown if you hover over the value on the search item.

My reasoning is that reading exact dates and times are tedious at a glance, and what you're really looking for is how far back in time something was - which we now calculate and present in way that is easier to skim.